RIOT-OS/RIOT
# VERSION: add 2020.01 version file

### Contribution description

This PR adds the VERSION of this release.

### Testing procedure

Version should be fine when compiling whatever application

```
make -C examples/hello-world/ all term
make: Entering directory '/home/francisco/workspace/RIOT/examples/hello-world'
Building application "hello-world" for "native" with MCU "native".

"make" -C /home/francisco/workspace/RIOT/boards/native
"make" -C /home/francisco/workspace/RIOT/boards/native/drivers
"make" -C /home/francisco/workspace/RIOT/core
"make" -C /home/francisco/workspace/RIOT/cpu/native
"make" -C /home/francisco/workspace/RIOT/cpu/native/periph
"make" -C /home/francisco/workspace/RIOT/cpu/native/stdio_native
"make" -C /home/francisco/workspace/RIOT/cpu/native/vfs
"make" -C /home/francisco/workspace/RIOT/drivers
"make" -C /home/francisco/workspace/RIOT/drivers/periph_common
"make" -C /home/francisco/workspace/RIOT/sys
"make" -C /home/francisco/workspace/RIOT/sys/auto_init
   text	   data	    bss	    dec	    hex	filename
  24299	    576	  47664	  72539	  11b5b	/home/francisco/workspace/RIOT/examples/hello-world/bin/native/hello-world.elf
/home/francisco/workspace/RIOT/examples/hello-world/bin/native/hello-world.elf  
RIOT native interrupts/signals initialized.
LED_RED_OFF
LED_GREEN_ON
RIOT native board initialized.
RIOT native hardware initialization complete.

main(): This is RIOT! (Version: 2020.01)
Hello World!
You are running RIOT on a(n) native board.
This board features a(n) native MCU.
```
